mix/CodeModel.cpp

```diff
@@ -188,3 +189,20 @@ void CodeModel::updateFormatting(QTextDocument* _document)
 {
 	m_result->codeHighlighter()->updateFormatting(_document, *m_codeHighlighterSettings);
 }
+
+dev::bytes const& CodeModel::getStdContractCode(const QString &_url)
+{
+	auto cached = m_compiledContracts.find(_url);
+	if (cached != m_compiledContracts.end())
+		return cached->second;
+
+	FileIo fileIo;
+	std::string source = fileIo.readFile(_url).toStdString();
+	solidity::CompilerStack cs;
+	cs.setSource(source);
```

Comment:
Note that if multiple contracts depend on each other, you have to `addSource(<identifier>, <content>)` them. Dependencies are resolved using import directives. But that is probably not yet in scope.